content/en/developers/marketplace/offering.md

```diff
@@ -0,0 +1,262 @@
+
+To set the location of the cloned repository directory: 
+
+```python
+ddev config set marketplace [/path/to/marketplace_directory]
+ddev config set repo marketplace
+```
+
+### Create an informational tile only listing
+
+For standalone SaaS licenses, Datadog Apps, professional services, and Datadog REST API-based integrations, the Datadog Development Toolkit creates the scaffolding for an informational tile-only listing. To create the informational tile-only listing's scaffolding, run `ddev create -t tile -v2 "<Offering Name>"` in the `marketplace` directory specified above.
+
+### Create a full data integration
+
+To generate a complete data integration's scaffolding, run `ddev create -v2 "<Offering Name>"` from the `marketplace` directory specified above.
+
+### Populate the tile scaffolding
+
+The `ddev` command generates a skeleton of the following folders and files belonging to your integration tile's assets.
+
+{{% integration-assets %}}
+
+#### README
+
+After creating a `README.md` file, follow these instructions to complete your listing:
+
+1. Write a description under an `## Overview` header that describes the value your offering provides to users and benefits to purchasing and installing the integration in the Datadog Marketplace (for example, out-of-the-box dashboards for more comprehensive observability, replays of user sessions, logs, alerts, and more). This information is displayed in the **Overview** tab on the integration tile.
+2. Include all the steps to setting up your Marketplace integration under a `##Setup` header that includes information divided into H3 headings (`###`) about installing the integration using the in-app integration tile, configuring the integration with the appropriate roles and permissions in your Datadog organization, out-of-the-box Datadog features that users who purchased and installed the integration can access (such as metrics, events, monitors, logs, dashboards, and more), and uninstalling the integration.
+3. Specify the types of data collected by your Marketplace integration under a `##Data Collected` header that includes information divided into H3 headings (`###`) about out-of-the-box metrics, events, and service checks. You can optionally include additional types of data collected such as logs, monitors, and dashboards. If your Marketplace integration does not provide these, use the following text in that particular section: `The [Datadog Marketplace] integration does not include any [events or service checks]`.
+4. At the end of the README, provide some contact information under a `###Support` header that includes an email to contact Support, a phone number to your company, or a link to your company's documentation or blog post.
+
+#### Media Carousel
+
+Listings created with the `-v2` flag allow you to implement a media carousel of images and a video into your integration tile. 
```

Comment:
```suggestion
A media carousel of images and a video is included in your integration tile. 
```
shouldn't be needed anymore since all new listings automatically have the v2 flag built in